docs/docs/cmd/outlook/message/message-remove.mdx

```diff
@@ -0,0 +1,55 @@
+import Global from '/docs/cmd/_global.mdx';
+
+# outlook message remove
+
+Removes a specifc message from a mailbox
+
+## Usage
+
+```sh
+m365 outlook message remove [options]
+```
+
+## Options
+
+```md definition-list
+`-i, --id <id>`
+: ID of the message.
+
+`--userId [userId]`
+: ID of the user that owns the mailbox. Specify either `userId` or `userName`, but not both. This option is required when using application permissions.
+
+`--userName [userName]`
+: User principal name of the user that owns the mailbox. Specify either `userId` or `userName`, but not both. This option is required when using application permissions.
+
+`-f, --force`
+: Don't prompt for confirmation.
+```
+
+<Global />
+
```

Comment:
In the specs, I've added a `remarks` section. Let's add this remark in the docs as well, maybe using an info admonition.